### PR TITLE
Add B2B step-up, OAuth tokens, and other fixes

### PR DESCRIPTION
**Pull request by jennifer-stytch**
This PR pulls in several new features, docs, and some fixes:
- Add B2B step-up
- Add Member OAuth tokens
- Fix RBAC implicit role assignment types
- Make email_id optional for OAuth auth factors
- Some docs fixes

### CODE REVIEW

File: src/b2b/organizations_members_oauth_providers.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    #[serde(with = "http_serde::status_code")]
+    pub status_code: http::StatusCode,
+    /// refresh_token: The `refresh_token` that you may use to obtain a new `access_token` for the User within
+    /// the provider's API.
+    pub refresh_token: std::option::Option<String>,
+}
+
+/// MicrosoftRequest: Request type for `OAuthProviders.google`, `OAuthProviders.microsoft`.
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct MicrosoftRequest {
```
Comment (logan-stytch) [inline src/b2b/organizations_members_oauth_providers.rs:46]: Request type for `google`? Sus...

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "stytch"
-version = "1.2.1"
+version = "1.3.0"
```
Comment (logan-stytch) [inline Cargo.toml:3]: tbh, there's enough in here to warrant a 2.0

1. PrimaryRequired moved from `b2b_discovery` to `b2b_sessions`
2. `email_id` type changed to `std::option::Option<string>` in many places
3. Overall, it's just a _big_ change with lots of new features

Let's not be afraid of bumping version numbers. I don't think anyone would be upset if we did a MAJOR update here and it also protects developers from cases where they think a MINOR wouldn't have any breaking changes.

Comment (jennifer-stytch) [reply Cargo.toml:3]: Sounds good, will change to major

**Review by logan-stytch — APPROVED** [review]

File: src/b2b/organizations_members_oauth_providers.rs (hunk 1 above)
Comment (jennifer-stytch) [reply src/b2b/organizations_members_oauth_providers.rs]: Thanks for updating codegen to handle this! Looks all better in [7798ba1](https://github.com/stytchauth/stytch-rust/pull/15/commits/7798ba164986803c6e0702267509188d2a88d63e)

**Pull request merged**